script.js

```diff
@@ -1,4 +1,83 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var dice = [];
+  var player1Score = [];
+  var player2Score = [];
+  var mode = 0;
+
+  // mode selection
+  if (input == 'LargeMode') {
+ 		var mode = 0;
```

Comment:
Can we use more descriptive names for modes? Why mode `0` an `1`? If you're going to use `0` and `1`, could we have comments to explain which mode is for what?